SwiftFlow/CoreTypes/Action.swift

```diff
@@ -7,19 +7,36 @@
 //
 
 import Foundation
-
+/**
+ This is Swift Flow's built in action type, it is the only built in type that conforms to the 
+ `Action` protocol. `StandardAction` can be serialized and can therefore be used with developer tools
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 101 characters (line_length)